File: Selection statement/Question6/main.java
import java.util.Scanner;

public class main {
    public static void main(String[] args) {
        try (Scanner scan = new Scanner(System.in)) {
            int num1, num2, num3;

            System.out.println("Enter first numbers: ");
            num1 = scan.nextInt();

            System.out.println("Enter second numbers: ");
            num2 = scan.nextInt();

            System.out.println("Enter third numbers: ");
            num3 = scan.nextInt();

            if (num1 > num2 && num1 > num3) {
                System.out.println("Largest number is: " + num1);
            } else if (num2 > num1 && num2 > num3) {
                System.out.println("Largest number is: " + num2);
            } else {
                System.out.println("Largest number is: " + num3);
            }
        }
    }
}

File: Selection statement/Question6/main.c
#include <stdio.h>

int main()
{
    int num1, num2, num3;

    printf("Enter three numbers: ");
    scanf("%d %d %d", &num1, &num2, &num3);

    if (num1 > num2 && num1 > num3)
    {
        printf("%d is the largest number\n", num1);
    }
    else if (num2 > num1 && num2 > num3)
    {
        printf("%d is the largest number\n", num2);
    }
    else if (num3 > num1 && num3 > num2)
    {
        printf("%d is the largest number\n", num3);
    }
    else if (num1 == num2 && num1 > num3)
    {
        printf("%d is the largest number\n", num1);
    }
    else if (num1 == num3 && num1 > num2)
    {
        printf("%d is the largest number\n", num1);
    }
    else if (num2 == num3 && num2 > num1)
    {
        printf("%d is the largest number\n", num2);
    }
    else
    {
        printf("All numbers are equal\n");
    }
    return 0;
}